RunJavaUtils.py

```diff
@@ -143,7 +143,7 @@ def Run_Java_File(destName, package, baseName):
             success = False
         try:
             os.remove(fullClassName)
-        except FileNotFoundError:
+        except os.FileNotFoundError:
```

Comment:
Is this correct? In my experiments, `FileNotFoundError` doesn't exist at all in Python 2 and it's a global (not in `os`) in Python 3.

Reply:
I'll change it back.  I'm still on Python 2, and PyCharm didn't like it, so I assumed it was in os.  So if me or someone else runs python 2 and hits that case, they'll get an unhandled exception, which is good enough for now.